callouts/python/extproc/example/ext_proc_client.py

```diff
@@ -72,7 +94,12 @@ def make_json_request(
     'prints out the responses.'
   )
   args = parser.parse_args()
-  # Preform callouts and print the responses.
-  logging.info(
-    list(make_json_request(json_list=args.data, address=args.address))
+  # Preform callouts and collect the responses.
+  responses = list(
+    make_json_request(json_list=args.data, address=args.address, key=args.key)
```

Comment:
can we go one step further as follows:
1. Have a pre-created dummy request with headers. This way, the user can simply run the client with just the server IP as the arg
2. Allow the user to specify the ProcessingRequest proto/json as an args

Reply:
1) Noted, I can add a dummy request in a followup 
2) Is that not what `-d` would be doing?